### DISCUSSION TITLE
Fichiers de config de la DB

### DISCUSSION THREAD

**Discussion by louthomas94** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#315] [Q&A] [answered]
Merci à vous!

**Comment by louthomas94** [comment]
Hello ! Je souhaite config les fichiers de config de la DB, quelqu'un pourrait m'aider ?

**Comment by Laikker** [comment]
Tu as suivi les étapes dans ici https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server?tab=readme-ov-file#prepare-and-configure-the-server ?

**Reply by louthomas94** [reply]
Yes j'ai bien suivi les étapes ! Mais au moment de faire migrer la DB ça ne marche pas

**Reply by Laikker** [reply]
"ca marche pas" c'est vraiment pas terrible pour décrire un problème, c'est à éviter quand on demande l'aide.
Décrit plutôt:
- les action que tu as effectuées, 
- les résultats que tu as obtenus avec copié/coller, screenshot si possible, 
- les résultats que tu aurais pensé obtenir

**Reply by louthomas94** [reply]
Yep sorry, j'ai donc installer MariaDB et créer ma base de donnée, mais je ne sais pas comment faire pour faire le fichier de config et l'intégrer au code.
J'ai lancer cette command "cd resources/[soz]/soz-core && yarn run prisma migrate deploy" et ça m'a dit cette erreur :
![image](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/114502065/86b88d02-e0b9-4806-9dc2-ffbeba8cdee1)

**Reply by Laikker** [reply]
Comme marqué dans la documentation, il faut modifier le resources/[soz]/soz-core/.env pour coller à ton environnement.
Si ton MariaDB tourne en local, il faut modifier soz_database par localhost par exemple.
Pense aussi à modifier aussi les crédentials

**Comment by Exaustia** [comment, marked as answer]
Hello,
CF message du dessus pour la configuration. 
Sinon, tu trouvera dans le global.cfg ligne 15, l'emplacement de ta config à set

**Reply by louthomas94** [reply]
Okay merci je vais look !
